editors/sc-ide/widgets/code_editor/sc_editor.cpp

```diff
@@ -1313,7 +1316,15 @@ void ScCodeEditor::findReferences()
 void ScCodeEditor::evaluateLine()
 {
     QString text;
-
+  
+    HelpBrowserDocklet* help = MainWindow::instance()->helpBrowserDocklet();
+    if (help) {
+        if (help->browser()->helpBrowserHasFocus()) {
```

Comment:
these `if`s could be combined, same below on line 1362 (and perhaps we could do a quick refactor of these methods in a new PR)